### DISCUSSION TITLE
How to achieve the effect

### DISCUSSION THREAD

**Discussion by Mankvis** [vuejs/vitepress#1966] [Q&A]
<img width="774" alt="image" src="https://user-images.githubusercontent.com/49258868/220234326-c41845d3-fb60-4f10-98f8-41b998106080.png">

Could you please tell me how to add such an outer box style to the code section in the theme

**Comment by xfq** [comment]
This is the current behaviour:

<img width="980" alt="h" src="https://user-images.githubusercontent.com/2863444/220257136-4ee36b3b-5561-4047-b885-2f3b75d36bc0.png">

What do you want to achieve?

**Comment by Mankvis** [comment]
<img width="786" alt="image" src="https://user-images.githubusercontent.com/49258868/220258500-e67cb962-ad93-48d3-b75f-cc76fad6be53.png">

Modify the style of the code container
